examples/resources/powerstore_protectionpolicy/protection_policy.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     powerstore = {
       version = "0.0.1"
-      source = "powerstore.com/powerstoreprovider/powerstore"
+      source = "dell/powerstore"
```

Comment:
I think source should be registry.terraform.io/dell/powerstore (As per the makefile)

You can verify once by running this .tf files

Reply:
registry.terraform.io is prefixed by default. However, added it to maintain code consistency.